tests/e2e/pact/specs/live/convert.successes.spec.ts

```diff
@@ -34,16 +35,32 @@ TestResources.convertCaseGroups.forEach(pactGroup => {
             const requestId = uuid.v4()
             const correlationId = uuid.v4()
 
+            // todo remove this skip for validation of dispense convert when we
+            // have a fhir dispense example which passes validation
+            const isDispenseNotification = 
+              (request.entry
+                .map(entry => entry.resource)
+                .filter(resource => resource.resourceType === "MessageHeader") as Array<MessageHeader>)[0]
+              .eventCoding?.code === "prescription-dispense"
+            const headers = isDispenseNotification 
+              ? {
+                "Content-Type": "application/fhir+json; fhirVersion=4.0",
+                "X-Request-ID": requestId,
+                "X-Correlation-ID": correlationId,
+                "x-skip-validation": "yepDoTheSkip" // presence of this header no matter its value will skip validation
```

Comment:
could just have 
```
const headers = {
  "Content-Type": "application/fhir+json; fhirVersion=4.0",
  "X-Request-ID": requestId,
  "X-Correlation-ID": correlationId
  "x-skip-validation": isDispenseNotification ? undefined : "yes"
}
```

Reply:
Thanks, didn't think of this. This workaround code has been removed now we have a fhir message which passes validation